### DISCUSSION TITLE
GitHub Archive Program snapshot for Arctic Code Vault

### DISCUSSION THREAD

**Discussion by qstlijku** [community/community#137917] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, not sure if this is the right place to ask but I recently read that a snapshot on February 2, 2020 of all active public repositories on GitHub were archived in the Arctic Code Vault.  Does anyone know if the data from this snapshot is easily accessible through any online libraries or GitHub Archive Program partners? Or if not, is any of it available upon request (e.g. for specific repositories)?

**Comment by AVMiraga** [comment]
Snapshots are only available through "Arctic Code Vault Contributor" badge owners. You can click on the badge and navigate through repositories. Limited to 3 repositories per badge.
